packages/amplify_flutter/pubspec.yaml

```diff
@@ -16,19 +16,48 @@ dependencies:
   amplify_storage_plugin_interface: 0.3.0-rc.2
   flutter:
     sdk: flutter
+  json_annotation: ^4.3.0
   meta: ^1.7.0
   plugin_platform_interface: ^2.0.0
 
+# dependency_overrides:
```

Comment:
If we intend to check out commented-out code, can we document why? Or is this an artifact?

Reply:
Added a comment.

Running builders with melos is impossible right now (you may have run into this). The only way to run `flutter pub run build_runner build` successfully is to use dependency overrides so that you can run `flutter pub get`.